tests/e2e/framework/kubernetes.go

```diff
@@ -147,21 +147,27 @@ type KubeInfo struct {
 
 func getClusterWideInstallFile() string {
 	const (
-		nonAuthInstallFile           = "istio.yaml"
-		authInstallFile              = "istio-auth.yaml"
-		nonAuthWithGalleyInstallFile = "istio-galley.yaml"
-		authWithGalleyInstallFile    = "istio-auth-galley.yaml"
+		nonAuthInstallFile                    = "istio.yaml"
+		authInstallFile                       = "istio-auth.yaml"
+		nonAuthWithGalleyInstallFile          = "istio-galley.yaml"
+		authWithGalleyInstallFile             = "istio-auth-galley.yaml"
+		nonAuthWithSidecarInjectorInstallFile = "istio-sidecar-injector.yaml"
+		authWithSidecarInjectorInstallFile    = "istio-auth-sidecar-injector.yaml"
 	)
 	var istioYaml string
 	if *authEnable {
 		if *useGalleyConfigValidator {
 			istioYaml = authWithGalleyInstallFile
+		} else if *useAutomaticInjection {
```

Comment:
This implies that galley and sidecarInjector are mutually exclusive but https://github.com/istio/istio/pull/6124/files#diff-d11449323523da124c3676e8b48e4597R34 enables both tests. I think we do _want_ to test both features at the same time in cluster wide tests, but we might need to refactor the helm setup files accordingly. Galley is now enabled by default in istio.yaml. We might be able to make a similar change in the e2e tests when clusterWide=true